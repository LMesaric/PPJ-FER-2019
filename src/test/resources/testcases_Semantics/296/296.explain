semantic: invalid escaped string 'bbb\x55' on byte offset 4
note: '"bbb\x55"' on line 18
