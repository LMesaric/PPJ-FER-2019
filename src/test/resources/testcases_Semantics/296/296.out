<primarni_izraz> ::= NIZ_ZNAKOVA(18,"bbb\x55")
